fiftyone/utils/flash.py

```diff
@@ -0,0 +1,140 @@
+        model: a ``flash.core.model.Task``
+        label_field ("predictions"): the name of the field in which to store
+            the model predictions. When performing inference on video frames,
+            the "frames." prefix is optional
+        confidence_thresh (None): an optional confidence threshold to apply to
+            any applicable labels generated by the model
+        store_logits (False): whether to store logits for the model
+            predictions. This is only supported when the provided ``model`` has
+            logits
+        batch_size (None): an optional batch size to use. Only applicable for
+            image samples
+        num_workers (None): the number of workers to use when loading images.
+            Only applicable for Torch models
+        skip_failures (True): whether to gracefully continue without raising an
+            error if predictions cannot be generated for a sample
+    """
+    if batch_size is not None:
+        print("Flash models only support the default batch size")
+
+    serializer = _get_fo_serializer(model, confidence_thresh, store_logits,)
+    with fou.SetAttributes(model, serializer=serializer):
+        predictions = model.predict(samples.values("filepath"))
+        for sample, prediction in zip(samples, predictions):
+            sample.add_labels(
+                prediction, label_field,
+            )
+
+
+def is_flash_model(model):
+    """Checks model type to determine if it is a flash model
+
+    Args:
+        model: the model instance to check
+    """
+    model_class = type(model)
+    mro = inspect.getmro(model_class)
+    for c in mro:
+        class_name = etau.get_class_name(c)
+        if class_name == "flash.core.model.Task":
+            return True
+
+    return False
+
+
+def normalize_labels(filepaths, predictions):
+    """Converts Detections from absolute to relative coordinates
+
+    Args:
+	filepaths: paths to the related images for every Detection used to generate metadata
+        predictions: the list of Detections to normalize 
+    """
+    for filepath, prediction in zip(filepaths, predictions):
+        metadata = ImageMetadata.build_for(filepath)
+        width = metadata.width
+        height = metadata.height
+        for detection in prediction.detections:
+            x, y, w, h = detection.bounding_box
+            detection.bounding_box = [
+                x / width,
+                y / height,
+                w / width,
+                h / height,
+            ]
+
+
+def _get_fo_serializer(model, confidence_thresh, store_logits):
+    """Initializes the FiftyOne serializer to be used for the given task"""
+    previous_serializer = model.serializer
+    if isinstance(model, fcc.ClassificationTask):
+        prev_args = dict(inspect.getmembers(model.serializer))
+        multi_label = False
+        if "multi_label" in prev_args:
+            multi_label = prev_args["multi_label"]
```

Comment:
`multi_label = prev_args.get("multi_label", False)` is a clean way to write these 3 lines